### DISCUSSION TITLE
Forcing UE TAU Request

### DISCUSSION THREAD

**Discussion by 3rt1m** [srsran/srsRAN_Project#54] [General Help]
Hello,

I am relatively new to LTE technology and I am trying to simulate a TAU request on a UE and wanted to know if this would be a way to achieve this.
Scenario:
Have 2 different srsran base stations (eNB-1 and eNB-2) within a RF screen room with different tac values each.
Both eNB-1 and eNB-2 have same earcfn and PLMN values to allow the UE to connect to them.
Have the UE device connect to eNB-1 and shortly after start the eNB-2.
Then turn off the eNB-1 to force the UE to do a handover to eNB-2.
I assume that the tac values being different will force the UE to perform a TAU request in this case.

Please let me know if this would work and if not what are some better options based on my setup?
Thanks!

**Comment by andrepuschmann** [comment]
Hey,
it seems you've used the wrong discussion board/repo to ask your question. All 4G related question should go the the srsRAN 4G repo - https://github.com/srsran/srsRAN_4G/discussions

But to answer your question anyway - what you describe is correct and a normal UE would behave like this. However, TAU isn't fully supported in srsUE so i would not send the TAU request.

**Reply by 3rt1m** [reply]
Thank you, I'll refer to the 4G discussion boards going forward.
